### PR TITLE
make the valid to the common function

### PR DESCRIPTION
**Pull request by chentao1596**
**What this PR does / why we need it**:

I think all the options check in the same func will be well, thank you.

```release-note
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
@chentao1596 
please add release note `NONE`. have a look at other PRs for examples if needed.

sgtm.
@kubernetes/sig-node-pr-reviews

**Comment by chentao1596** [comment]
@neolit123 @nikhita thank you, added it.

File: cmd/kubelet/app/options/options.go [hunk 1]
```diff
@@ -250,6 +250,10 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 	if f.NodeStatusMaxImages < -1 {
 		return fmt.Errorf("invalid configuration: NodeStatusMaxImages (--node-status-max-images) must be -1 or greater")
 	}
+	// Obtain Kubelet Lock File
+	if f.ExitOnLockContention && f.LockFilePath == "" {
+		return fmt.Errorf("cannot exit on lock file contention: no lock file specified")
```
Comment (mtaufen) [inline cmd/kubelet/app/options/options.go]: Please change the error message to `invalid configuration: when ExitOnLockContention (--exit-on-lock-contention) is true, LockFilePath (--lock-file) must be non-empty`, so it is consistent with the rest of the validation errors.

Comment (chentao1596) [reply cmd/kubelet/app/options/options.go]: it's done, please help to see again, thank you!

**Comment by mtaufen** [comment]
Looks good :)
Please squash your commits together and then I will LGTM.

**Comment by chentao1596** [comment]
@mtaufen Squashed, thank you for your review, ^_^!

**Comment by yujuhong** [comment]
/approve

based on @mtaufen's review

**Comment by mtaufen** [comment]
Please rebase

**Pull request closed without merging**